app/models/admin.rb

```diff
@@ -0,0 +1,4 @@
+class Admin < ActiveRecord::Base
+  validates :username, presence: true
```

Comment:
Perhaps add a uniqueness one here? Do we need it? Also, which admin does this refer to? Is this the super admin (the site's admin?) or a job poster?